tests/integration_tests.py

```diff
@@ -40,5 +47,82 @@ def preparacion_cuentas_naranja_integration_test():
+    expected_df_result_5 = pd.read_csv(result_directory_path / 'fake_subcliente_5.csv', encoding='latin-1', sep=';')
+    expected_df_result_6 = pd.read_csv(result_directory_path / 'fake_subcliente_6.csv', encoding='latin-1', sep=';')
+    expected_df_result_7 = pd.read_csv(result_directory_path / 'fake_subcliente_7.csv', encoding='latin-1', sep=';')
+    expected_df_result_8 = pd.read_csv(result_directory_path / 'fake_subcliente_8.csv', encoding='latin-1', sep=';')
+    expected_df_result_9 = pd.read_csv(result_directory_path / 'fake_subcliente_9.csv', encoding='latin-1', sep=';')
+    expected_df_result_10 = pd.read_csv(result_directory_path / 'fake_subcliente_10.csv', encoding='latin-1', sep=';')
+
+    with mock.patch('builtins.input', return_value='fake'):
+        result_directory_path = Preparacion_Cuentas_Comafi(emerix_file_path=emerix_test_file_path)
+    result_directory_path = Path(result_directory_path)
+
+    files_in_directory_result = os.listdir(result_directory_path)
+
+    assert sorted(expected_files_in_result_directory) == sorted(files_in_directory_result)
+
+    for result_path in files_in_directory_result:
+        if 'fake_subcliente_0.csv' in result_path:
+            df_result_0 = pd.read_csv(result_directory_path / result_path, encoding='latin-1', sep=';')
+        elif 'fake_subcliente_1.csv' in result_path:
+            df_result_1 = pd.read_csv(result_directory_path / result_path, encoding='latin-1', sep=';')
+        elif 'fake_subcliente_2.csv' in result_path:
+            df_result_2 = pd.read_csv(result_directory_path / result_path, encoding='latin-1', sep=';')
+        elif 'fake_subcliente_3.csv' in result_path:
+            df_result_3 = pd.read_csv(result_directory_path / result_path, encoding='latin-1', sep=';')
+        elif 'fake_subcliente_4.csv' in result_path:
+            df_result_4 = pd.read_csv(result_directory_path / result_path, encoding='latin-1', sep=';')
+        elif 'fake_subcliente_5.csv' in result_path:
+            df_result_5 = pd.read_csv(result_directory_path / result_path, encoding='latin-1', sep=';')
+        elif 'fake_subcliente_6.csv' in result_path:
+            df_result_6 = pd.read_csv(result_directory_path / result_path, encoding='latin-1', sep=';')
+        elif 'fake_subcliente_7.csv' in result_path:
+            df_result_7 = pd.read_csv(result_directory_path / result_path, encoding='latin-1', sep=';')
+        elif 'fake_subcliente_8.csv' in result_path:
+            df_result_8 = pd.read_csv(result_directory_path / result_path, encoding='latin-1', sep=';')
+        elif 'fake_subcliente_9.csv' in result_path:
+            df_result_9 = pd.read_csv(result_directory_path / result_path, encoding='latin-1', sep=';')
+        elif 'fake_subcliente_10.csv' in result_path:
+            df_result_10 = pd.read_csv(result_directory_path / result_path, encoding='latin-1', sep=';')
```

Comment:
Yes, I almost use a for loop, but I recalled that it could be bad practice to use it in a test, because, who tests then the test jajaj. I wonder if parameterized has a decorator to not run the same test several times instead, run a part of it, but is the same as a loop.

The other approach I'm thinking of now is using a custom `set_up` method for each integration test. Making the command an creating the result, and then in the test, testing every result with parameterized